ansible/playbooks/AdoptOpenJDK_Unix_Playbook/group_vars/all/adoptopenjdk_variables.yml

```diff
@@ -32,3 +32,12 @@ Vendor_Playbook: /Vendor_Files/Vendor_Playbook/Vendor.yml
 
 # Default BootJDK installed
 bootjdk: hotspot
+
+# GPG Public Keys
+key:
```

Comment:
Did these keys just come from the messages you get when you try to verify one without the signature being present, or were they from the upstream project's somwhere?

Reply:
Some had the key ids present in the same place as the binary to download. For the others I did find their fingerprint in the inital verification message. But for either case, I did search for the key on https://keyserver.ubuntu.com/ to verify its authenticity. Is this the best way to have gone about this?